subconscious/column.py

```diff
@@ -0,0 +1,18 @@
+from .model import Column
+
+
+class Integer(Column):
+    def __init__(
+            self,
+            primary_key=None,
+            composite_key=None,
+            index=None,
+            required=None,
+            enum=None,
+            sort=None, auto=True):
+        super(Integer, self).__init__(int, primary_key, composite_key, index, required, enum, sort)
```

Comment:
please use keyword arguments for this. Too easy to swap the order of two fields and make a big mistake.